README.md

```diff
@@ -15,7 +15,7 @@ repositories {
 
 dependencies {
      (other dependencies)
```

Comment:
Should we comment the first line just to make it super super clear that `(other dependencies)` isn't an actual piece of code?

Reply:
Sure, don't see why not